### PR TITLE
Add related streaming version of this module to the readme

### LINKED ISSUE #13: Support streams
Would be nice to remove ansi from streams, for example for people wanting to pipe from stdout to a file without the mess of colour codes.

### PR DESCRIPTION
**Pull request by kevva**
Fixes #13.

### CODE REVIEW

**Comment by shinnn** [comment]
IMO we can achieve https://github.com/chalk/strip-ansi/issues/13 by just adding `replacestream` example in the README.

In most case users would not need `.stream()`, while they unexpectedly installs unused `replacestream` dependency.

**Comment by kevva** [comment]
Probably better, I'll update the PR.

File: readme.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ stripAnsi('\u001B[4mUnicorn\u001B[0m');
 //=> 'Unicorn'
 ```
 
+If you want to support streams you can use [`replacestream`](https://github.com/eugeneware/replacestream) like below:
+
+```js
+const fs = require('fs');
+const replacestream = require('replacestream');
+const stripAnsi = require('strip-ansi');
```
Comment (shinnn) [inline readme.md]: You meant,

```js
const ansiRegex = require('ansi-regex');
```

Comment (kevva) [reply readme.md]: Oh, yeah. Should it really be documented here then?

**Comment by shinnn** [comment]
I'm rethinking my suggestion "just adding `replacestream` example to *strip-ansi's* README" – stream example doesn't require `strip-ansi` itself but uses `ansi-regex` directly. So, as @kevva said in https://github.com/chalk/strip-ansi/pull/15#discussion_r139714325, it's a bit confusing and should be documented in the `ansi-regex` README instead.

Another alternative is to create a new separate module `strip-ansi-stream` and refer to it in strip-ansi's README.

Anyway, it's better to wait for opinions from Chalk team and @popey456963.

**Comment by sindresorhus** [comment]
Let's do it as a separate module: https://github.com/chalk/strip-ansi-stream (@kevva Wanna push your code there?)

**Comment by kevva** [comment]
Done https://github.com/chalk/strip-ansi-stream/commit/ca27b268a5d3e5981a22060606dbf65d1a8b5190. I'll just update this PR with a link in the related section.

**Comment by sindresorhus** [comment]
Great! @kevva Can you also add it to https://github.com/chalk/chalk#related ?

**Pull request merged**